de.html

```diff
@@ -1868,66 +1867,67 @@
 		<a target="_blank" href="http://augmentingcognition.com/ltm.html">
 		Augmenting Long-Term Memory</a>
 		by Michael Nielsen
-		showed me that Spaced Repetition wasn't just a tool for memorization,
-		it's a tool to build deep understanding. It could even be a way of <i>life</i>.
+		hat mir gezeigt, dass Spaced Repetition nicht nur ein Werkzeug zum Auswendiglernen ist,
+		sondern ein Werkzeug, um ein tieferes Verständnis aufzubauen. Es könnte sogar eine
+		Lebenseinstellung sein.
 	</li>
 	<li>
 		💬
 		<a target="_blank" href="https://fluent-forever.com/the-book/">
 		Fluent Forever</a>
 		by Gabriel Wyner
-		convinced me to finally (re)start learning French,
-		and create a Leitner Box. (my 64-day calendar was adapted from this book)
+		überzeugte mich endlich (wieder) Französisch zu lernen und eine Leinter Box zu erstellen
+		(Mein 64-Tage-Kalebder wurde von diesem Buch übernommen).
 	</li>
 	<li>
 		🃏
 		<a target="_blank" href="https://www.supermemo.com/en/articles/20rules">
 		20 Rules for Cards</a>
-		by Piotr Wozniak taught me how to squeeze the most out of Spaced Repetition.
-		(Note: the author also invented the algorithm used by Anki!)
+		by Piotr Wozniak hat mir beigebracht, wie ich das Beste aus Spaced Repetition herausholen kann.
+		(Hinweis: Der Autor hat auch den von Anki verwendeten Algorithmus erfunden!)
 	</li>
 </ul>
 
-	And my favorite reads on the science of learning:
+	Und meine Lieblingslesungen über die Wissenschaft des Lernens:
 
 <ul>
 	<li>
 		👩‍🎓
 		<a target="_blank" href="https://www.amazon.com/Why-Dont-Students-Like-School/dp/047059196X/">
 		Why Don't Students Like School?</a>
 		by Daniel Willingham
-		proved to me that "mere memorization" is actually <i>necessary</i>
-		for creativity &amp; critical thinking.
+		hat mir bewiesen, dass "bloßes Auswendiglernen" tatsächlich notwendig ist für Kreativität
+		&amp; kritisches Denken.
 	</li>
 	<li>
 		✏️
 		<a target="_blank" href="http://tlcp.depaultla.org/wp-content/uploads/sites/2/2015/05/Dunlosky-et-al-2013-What-Works-What-Doesnt.pdf">
 		What Works, What Doesn't [PDF]
 		</a>
 		by Dunlosky et al
-		summarizes <i>200+ studies</i> on various learning methods.
-		In sum: common techniques like re-reading &amp; highlighting don't work,
-		less-common techniques like spacing &amp; self-testing do.
+		fasst mehr als 200 Studien zu verschiedenen Lernmethoden zusammen.
+		In der Summe: Gängige Techniken, wie erneutes Lesen &amp; Hervorheben funktionieren nicht so gut,
+		wie es weniger verbreitete Techniken, wie Abstand &amp; Selbsttest tun.
```

Comment:
Ist hier im Original mit "spacing" "Abstand" oder "spaced repitition" gemeint?

Reply:
Ich glaube auch, dass es hier eher um "spaced repetition" geht. 
```suggestion
		wie es weniger verbreitete Techniken wie "spaced repitition" &amp; Selbsttest tun.
```